Jetcaster/wear/src/main/java/com/example/jetcaster/ui/queue/QueueScreen.kt

```diff
@@ -83,30 +81,33 @@ fun QueueScreen(
     uiState: QueueScreenState,
     onPlayButtonClick: () -> Unit,
     onPlayEpisodes: (List<PlayerEpisode>) -> Unit,
-    modifier: Modifier = Modifier,
     onEpisodeItemClick: (PlayerEpisode) -> Unit,
     onDeleteQueueEpisodes: () -> Unit,
     onDismiss: () -> Unit,
+    modifier: Modifier = Modifier,
 ) {
-    val columnState = rememberResponsiveColumnState(
-        contentPadding = padding(
-            first = ScalingLazyColumnDefaults.ItemType.Text,
-            last = ScalingLazyColumnDefaults.ItemType.Chip,
-        ),
+    val contentPadding = rememberResponsiveColumnPadding(
+        first = ColumnItemType.ListHeader,
+        last = ColumnItemType.Button,
     )
+
+    val columnState = rememberTransformingLazyColumnState()
     ScreenScaffold(
         scrollState = columnState,
+        contentPadding = contentPadding,
         modifier = modifier,
-    ) {
+    ) { contentPadding ->
         when (uiState) {
             is QueueScreenState.Loaded -> QueueScreenLoaded(
```

Comment:
what does the is do

Reply:
similar as above, https://developer.android.com/training/wearables/compose/screen-size#responsive-layouts. This is a best practice well known on Wear OS development.